lib/get-repo-pipeline-manager.js

```diff
@@ -72,6 +72,7 @@ export default function({confirm, notificationManager, workspace}) {
       return result;
     } catch (error) {
       if (error instanceof GitError) {
+        repository.didPullError();
```

Comment:
What's the reason for generalizing this from specifically merge errors to any pull errors?